app/views/my/index.html.erb

```diff
@@ -10,7 +10,7 @@
 
 <% @page_title = "#{current_tab.capitalize} listing" %>
 
-<h1 class="sr-only">My Files, Collections, Highlights, and Files Shared with Me</h1>
+<h1 class="sr-only">My Works, Collections, Highlights, and Files Shared with Me</h1>
```

Comment:
Should we also `s/Files Shared with Me/Works Shared with Me/g` or should that be a separate PR?